### PR TITLE
Update an outdated CLI for the arithmetics example

### PR DESCRIPTION
**Pull request by Pluralia**
The current CLI version is corresponding to [CLI for domainmodel](https://github.com/langium/langium/pull/160) and [CLI for statemachine](https://github.com/langium/langium/pull/171).

### CODE REVIEW

File: examples/arithmetics/README.md [hunk 1]
```diff
@@ -4,8 +4,11 @@
 
 The Arithmetics Example features an interpreter that you can run via cli.
 
-Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
-Use `node ./bin/cli` to run the cli. Follow the instructions or use `node ./bin/cli eval <full-path-to-calc-file>`.
+* Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
+* Run `npm install -g ./` from the arithmetics directory.
+* Use `arithmetics-cli` to run the cli. Follow the instructions or use `arithmetics-cli eval <full-path-to-calc-file>`.
```
Comment (spoenemann) [inline examples/arithmetics/README.md:9]: This is one way to run it, but for quick testing I find it easier to just call  `./bin/cli`. Can you add this as an alternative to the global install (affects the other examples, too)?

Comment (Pluralia) [reply examples/arithmetics/README.md]: I've updated the arithmetics README -- could you check? If it's okay, I update all other READMEs.

File: examples/arithmetics/README.md [hunk 2]
```diff
@@ -4,8 +4,14 @@
 
 The Arithmetics Example features an interpreter that you can run via cli.
 
-Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
-Use `node ./bin/cli` to run the cli. Follow the instructions or use `node ./bin/cli eval <full-path-to-calc-file>`.
+* Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
+* Use `node ./bin/cli` from the arithmetics directory to run the cli. Follow the instructions or use `node ./bin/cli eval <full-path-to-calc-file>`.
+
+The interpreter calculates each Evaluation in the source file and prints result.
```
Comment (spoenemann) [inline examples/arithmetics/README.md:10]:
```suggestion
The interpreter calculates each Evaluation in the source file and prints the result.
```

File: examples/arithmetics/README.md [hunk 3]
```diff
@@ -4,8 +4,14 @@
...
+
+You also can use `arithmetics-cli` as a replacement for `node ./bin/cli`, if install the cli globally.
```
Comment (spoenemann) [inline examples/arithmetics/README.md:12]:
```suggestion
You also can use `arithmetics-cli` as a replacement for `node ./bin/cli`, if you install the cli globally:
```

**Review by spoenemann — COMMENTED** [review]
Yes, please update the other example READMEs accordingly.

**Review by spoenemann — APPROVED** [review]
Thanks!

**Pull request merged**